tests/integration/raw_studies_blueprint/test_fetch_raw_data.py

```diff
@@ -297,206 +298,405 @@ def test_get_study(self, client: TestClient, user_access_token: str, internal_st
-    res = client.put(raw_url, params={"path": existing_folder, **additional_params})
-    _check_endpoint_response(study_type, res, client, internal_study_id, expected_msg, "FolderCreationNotAllowed")
-
-
-def test_retrieve_from_archive(client: TestClient, user_access_token: str) -> None:
-    # client headers
-    client.headers = {"Authorization": f"Bearer {user_access_token}"}
-
-    # create a new study
-    res = client.post("/v1/studies?name=MyStudy")
-    assert res.status_code == 201
-
-    # get the study id
-    study_id = res.json()
-
-    # add a new area to the study
-    res = client.post(
-        f"/v1/studies/{study_id}/areas",
-        json={
-            "name": "area 1",
-            "type": "AREA",
-            "metadata": {"country": "FR", "tags": ["a"]},
-        },
-    )
-    assert res.status_code == 200, res.json()
-
-    # archive the study
-    res = client.put(f"/v1/studies/{study_id}/archive")
-    assert res.status_code == 200
-    task_id = res.json()
-    wait_for(
-        lambda: client.get(
-            f"/v1/tasks/{task_id}",
-        ).json()["status"]
-        == 3
-    )
-
-    # retrieve a `Desktop.ini` file from inside the archive
-    rel_path = "Desktop"
-    res = client.get(
-        f"/v1/studies/{study_id}/raw",
-        params={"path": rel_path, "formatted": True},
-    )
-    assert res.status_code == 200
-
-    # retrieve a `study.antares` file from inside the archive
-    rel_path = "study"
-    res = client.get(
-        f"/v1/studies/{study_id}/raw",
-        params={"path": rel_path, "formatted": True},
-    )
-    assert res.status_code == 200
+        res.raise_for_status()
+
+        # Create a cluster in the first area
+        res = client.post(
+            f"/v1/studies/{study_id}/areas/{area1_id}/clusters/thermal",
+            json={"name": cluster_id.title(), "group": "Nuclear"},
+        )
+        res.raise_for_status()
+
+        # create a binding constraint that references the link
+        bc_id = "bc_1"
+        bc_obj = {
+            "name": bc_id,
+            "enabled": True,
+            "time_step": "daily",
+            "operator": "less",
+            "terms": [
+                {
+                    # Cluster in an area
+                    "data": {"area": area1_id, "cluster": cluster_id.lower()},
+                    "id": f"{area1_id}.{cluster_id.lower()}",
+                    "offset": 2,
+                    "weight": 1.0,
+                }
+            ],
+        }
+        res = client.post(f"/v1/studies/{study_id}/bindingconstraints", json=bc_obj)
+        res.raise_for_status()
+
+        # retrieve the binding constraint `bc_1` less matrix
+        rel_path = "input/bindingconstraints/bc_1_lt"
+        res = client.get(
+            f"v1/studies/{study_id}/raw/original-file",
+            params={"path": rel_path},
+        )
+        res.raise_for_status()
+
+        # skip the test if the OS is Windows
```

Comment:
You shouldn't skip this test on windows

Reply:
I thought we already talked about this and agreed to ignore it for Windows as it fails for a link in an archive.